### ISSUE TITLE
[Feature Request] Cop for optional hash braces

### ISSUE THREAD

**Issue by ahawkins** [rubocop/rubocop#1137]
The community does not use `{}` on a final hash argument. Here's an example: `select "Bump Up", { from: "Products" }`. Rubocop should catch this violation. It should be written as `select "Bump Up", from: "Products"`.

**Comment by developerinlondon** [comment]
so what should code as above be written as?

**Comment by mikegee** [comment]
@developerinlondon, `Style/BracesAroundHashParameters` wants that example code to look like this:

```ruby
some_method something, a: 1, b: 2
```
